src/components/tag/index.md

```diff
@@ -39,6 +45,8 @@ Tags should be helpful to users. The more you add, the harder it is for users to
 
 {{ example({ group: "components", item: "tag", example: "coloured-tags", html: true, nunjucks: true, open: false }) }}
 
+<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Any tag implementation that allows you to change the order of tags must offer a way to do so without relying on ‘click and drag’ movements. This is to comply with WCAG 2.2 success criterion [2.5.7 Dragging Movements](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html).
```

Comment:
```suggestion
<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Any tag implementation that allows the user to change the order of tags must offer a way to do so without relying on ‘click and drag’ movements. This is to comply with WCAG 2.2 success criterion [2.5.7 Dragging Movements](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements.html).
```

Reply:
Should have been changed in more recent commits